src/components/DropZone.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import '../styles/DropZone.scss'
+
+const DropZone = props => {
+  const { onDrop, onDragLeave, channelName } = props
```

Comment:
```suggestion
  const { onDrop, onDragLeave, label } = props
```

Reply:
DropZone component could be more reusable if _channelName_ prop would be just a _label_. The label could also then be translated in the parent component.